src/me/components/AlertsActivity.tsx

```diff
@@ -0,0 +1,67 @@
+// Libraries
+import React, {createContext, FC, useMemo} from 'react'
+
+// Components
+import {Panel, Heading, HeadingElement, FontWeight} from '@influxdata/clockface'
+import {runAlertsActivityQuery} from 'src/alerting/utils/activity'
+import {useSelector} from 'react-redux'
+import EventViewer from 'src/eventViewer/components/EventViewer'
+import EventTable from 'src/eventViewer/components/EventTable'
+import {ResourceType} from 'src/types'
+import GetResources from 'src/resources/components/GetResources'
+import {getCheckIDs} from 'src/checks/selectors'
+import {Fields} from 'src/eventViewer/types'
+import {getOrg} from 'src/organizations/selectors'
+
+interface Props {
+  fields: Fields
+}
+
+export const CheckIDsContext = createContext<{[x: string]: boolean}>(null)
+
+const AlertsActivity: FC<Props> = ({fields}) => {
+  const {id: orgID} = useSelector(getOrg)
+  const checkIDs = useSelector(getCheckIDs)
+
+  const loadRows = useMemo(
+    () => options => runAlertsActivityQuery(orgID, options),
```

Comment:
question: is it possible that we would want to execute `runAlertsActivityQuery` a second time even if the `orgID` hasn't changed? Please correct me if I'm wrong but `useMemo` in this case would only execute `runAlertsActivityQuery` a 2nd time if `orgID` changes, right? I'm just wondering if the status information of the alerts might have changed and perhaps we should pull it more frequently?